translations/fr/01-introduction.md

```diff
@@ -1,73 +1,61 @@
-1. **La sauce secrète:** Ce chapitre vous dévoilera les stratégies et les techniques pour démarrer, gagner du terrain dans vos contributions et développer votre CV open source.
-1. **Conclusion:** Nous conclurons le cours avec des points clés et discuterons des prochaines étapes de votre parcours open source.
+1. **Introduction :** Nous commencerons par préparer le terrain pour votre parcours open source, en discutant des prérequis, des objectifs et des attentes.
+2. **Qu'est-ce que l'open source ? :** Dans ce chapitre, nous explorerons la définition de l'open source et son histoire, vous fournissant une base solide pour comprendre le contexte de la collaboration open source.
+3. **Pourquoi l'Open Source ? :** Nous aborderons les avantages de l'Open Source, notamment l'accès aux connaissances, la communauté et le développement des compétences.
+4. **Outils pour réussir :** Ce chapitre vous présentera les outils essentiels pour une collaboration open source réussie, notamment GitHub, Git, CodeSee, VS Code et Discord.
+5. **Comment contribuer à l'Open Source :** Dans ce chapitre, nous discuterons du processus de recherche de projets sur lesquels travailler, de soumission de contributions, d'intégration dans un nouveau projet et de ce à quoi s'attendre après la soumission de votre contribution.
+6. **La sauce secrète :** Ce chapitre vous dévoilera les stratégies et les techniques pour démarrer, gagner du terrain dans vos contributions et développer votre CV open source.
+7. **Conclusion :** Nous conclurons le cours avec des points clés et discuterons des prochaines étapes de votre parcours open source.
 
-Tout au long de ce cours, nous fournirons des exemples et des démonstrations pour aider à illustrer les concepts clés et fournir des opportunités pratiques de pratique.
-Dans la mesure du possible, nous présenterons des projets et des outils open source réels, tels que [OpenSauced](https://opensauced.pizza/), pour montrer comment appliquer les compétences que vous avez acquises.
+Tout au long de ce cours, nous fournirons des exemples et des démonstrations pour aider à illustrer les concepts clés et fournir des opportunités pratiques de pratique. Dans la mesure du possible, nous présenterons des projets et des outils open source réels, tels que [OpenSauced](https://opensauced.pizza/), pour montrer comment appliquer les compétences que vous avez acquises.
 
 Veuillez noter que bien que ces exemples et démos soient conçus pour vous guider et vous inspirer, votre propre parcours open source peut différer en fonction de vos intérêts, de vos compétences et des projets choisis.
 
 Utilisez les exemples et les démonstrations comme point de départ, mais n'ayez pas peur d'explorer d'autres projets, outils et ressources à mesure que vous continuez à apprendre et à grandir.
 
 ## LFG 🚀
```

Comment:
We've updated this to "Let's Get Started 🚀"

Reply:
I forgot to translate this one, good catch!